VocaDbWeb/Scripts/Components/Shared/Partials/User/ProfileIconKnockout.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+interface ProfileIconKnockoutProps {
+	icon?: string;
+	size: number;
+}
+
+const ProfileIconKnockout = React.memo(
+	({ icon, size }: ProfileIconKnockoutProps): React.ReactElement => {
+		return (
+			<div
+				style={{
+					width: `${size}px`,
+					height: `${size}px`,
+					display: 'inline-block',
+				}}
+			>
+				<img src={icon ?? '/Content/unknown.png'} alt="User avatar" />
+			</div>

```

Comment:
[In the original implementation](https://github.com/VocaDB/vocadb/blob/585136f696bdff161acbd3c6e6599860a66d8386/VocaDbWeb/Views/Shared/Partials/User/_ProfileIconKnockout.cshtml#L4), if a user doesn't have an email address associated with their account, the div element will be hidden.